### DISCUSSION TITLE
Edrumulus with LCD and buttons without PC

### DISCUSSION THREAD

**Discussion by ojungleboy** [corrados/edrumulus#68] [Ideas]
Hello everyone, as long as we now have an MIDI circuit. I would like to know if it's possible to use Edrumulus without PC (Octave software). Instead, to add a LCD display and some navigation buttons so we can get rid of the computer and use Edrumulus straight with MIDI interface.

Thank you so much and keep it up the hardwork

**Comment by corrados** [comment]
Do you intend to use Edrumulus together with a PC running [Superior Drummer](https://www.toontrack.com/product/superior-drummer-3/) or similar software? In that case you could run simple software tool to configure Edrumulus via MIDI.

**Reply by ojungleboy** [reply]
> Do you intend to use Edrumulus together with a PC running [Superior Drummer](https://www.toontrack.com/product/superior-drummer-3/) or similar software? In that case you could run simple software tool to configure Edrumulus via MIDI.

Thanks for your reply.

I will use it through MIDI triggering, such smartwav2 or a raspberry pi 4 running samplebox.

The idea is to get rid of the PC on the system.

So, it would be great if I could change settings such note number, sensitivity, threshold, etc using buttons and LCD display

**Reply by corrados** [reply]
If you are using a Raspberry Pi, you might be interested in the [button/display PCB ](https://github.com/corrados/edrumulus/discussions/58#discussioncomment-3478121)I created for prototype 5.

**Reply by thijstriemstra** [reply]
>  if you are using a Raspberry Pi

These boards are very expensive for a long time now, and unfortunately they're not an option because of that.

**Reply by corrados** [reply]
Fortunately, now they are available again: https://rpilocator.com
